### PR TITLE
Cleanup main repo

### PR DESCRIPTION
**Pull request by rokroskar**
Removing things we are not using anymore, mostly with regards to our (now obsolete) docker-compose setup and even some other relics from the distant past. 

extra bonus: the `run-tests` script can now again be used to test things locally 

Note that I removed `shellcheck` -- we can re-introduce it to check the scripts inside `./scripts` but I would then make a separate issue/PR for fixing up all the shell scripts as it looks to be a bit of work.

### CODE REVIEW

**Comment by leafty** [comment]
Is it still a WIP?

**Comment by rokroskar** [comment]
@leafty I don't think so -- I'll make sure it deploys ok still but the tests pass (I was expecting more of a struggle there, hence the `[WIP]`)

File: docs/developer/minikube.rst [hunk 1]
```diff
@@ -100,17 +100,7 @@ For more on the Renku helm charts, go to ``charts/renku/README.rst``.
...
-.. code-block:: console
-
-    $ kubectl -n renku run renku-demo -it \
-        --env="GITLAB_URL=http://$(minikube ip)/gitlab" \
-        --env="KEYCLOAK_URL=http://$(minikube ip)" \
-        --image=renku/renku-demo:latest \
-        --restart=Never
-
-Then we can run the tests:
+Run the tests with ``helm``:
```
Comment (leafty) [inline docs/developer/minikube.rst]: No need to run the demo script now?

**Comment by leafty** [comment]
I'm glad I can read the `Makefile` again!

File: docs/developer/minikube.rst (hunk 1 above)
Comment (rokroskar) [reply docs/developer/minikube.rst]: I moved it to the test init container.

**Comment by rokroskar** [comment]
@leafty actually I think we could remove the makefile completely. That requires adding some more instructions to the various docs so the lazy option is to leave it be for now.

**Pull request merged**